src/lib/get-participations.ts

```diff
@@ -0,0 +1,20 @@
+import type { Participations } from '@guardian/ab-core';
+import { isObject, isString, storage } from '@guardian/libs';
+
+const participationsKey = 'gu.ab._participations';
```

Comment:
Was there always an `_` in the key for this? Wasn't this originally `gu.ab.participations`?

Reply:
It seems to differ between frontend and DCR:

https://github.com/guardian/dotcom-rendering/pull/6136#issuecomment-1270495167

Reply:
I don't understand why, or which we should care about here. Will do some digging to better understand how participations are stored. Thanks for this prompt!

Reply:
As I understand it, Frontend used `gu.ab.participations` and saves a participation in local storage whenever a browser is entered into a test, and this logic wasn't present on DCR. This logic was added to DCR for the long Mandatory Sign In Gate test in order to bucket people and then stop adding new people to buckets after a certain duration. However, I think because of a naming collision this needed to be called something other than `gu.ab.participations`, hence the underscore.

I think from a commercial perspective the only tests we're interesting in targeting against are the traditional Frontend ones, so `gu.ab.participations`. I'm not sure that the `_` ones are ever exposed to the commercial code.

Reply:
Great, thanks Chris, that was the conclusion I just reached too. I've updated to use `gu.ab.participations`